boards/nxp/mimxrt1170_evk/doc/index.rst

```diff
@@ -168,6 +168,9 @@ RT1170 EVKB (`mimxrt1170_evk@B//cm7/cm4`)
 +-----------+------------+-------------------------------------+-----------------+-----------------+
 | PIT       | on-chip    | pit                                 | Supported (M7)  | Supported (M7)  |
 +-----------+------------+-------------------------------------+-----------------+-----------------+
+| VIDEO     | on-chip    | CSI; MIPI CSI-2 Rx. Tested with     | Supported (M7)  | Supported (M7)  |
```

Comment:
This can probably be in the commit where we add video support to the YAML for the RT1170 EVK

Reply:
Okay, I will merge these two commits.